diff --git a/src/documents/file_handling.py b/src/documents/file_handling.py
index 3d1a643..81a863d 100755
--- a/src/documents/file_handling.py
+++ b/src/documents/file_handling.py
@@ -3,6 +3,12 @@ import os
 from django.conf import settings

 from documents.models import Document
+import locale
+
+locale_setting = os.getenv('LC_ALL')
+if locale_setting is None:
+ locale_setting = 'en_US.UTF8'
+locale.setlocale(locale.LC_ALL, locale_setting)
 from documents.templating.filepath import validate_filepath_template_and_render
 from documents.templating.utils import convert_format_str_to_template_format

